integration-tests/testconfig/vrfv2plus/vrfv2plus.toml

```diff
@@ -9,6 +9,9 @@ use_existing_env = false
 subscription_funding_amount_link = 5.0
 subscription_funding_amount_native=1
 
+subscription_refunding_amount_link = 5.0
```

Comment:
Would it make sense to create TOML config examples per chain or would that be too much to maintain?